diff --git a/src/components/fields/ObjectField.tsx b/src/components/fields/ObjectField.tsx
index 8399429da5d89bec13da4150c6053731a1b95071..b602b526d11ea7843d056dab164d9872a5ee3714 100644
--- a/src/components/fields/ObjectField.tsx
+++ b/src/components/fields/ObjectField.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { Component } from 'react';
 import {
   getTemplate,
diff --git a/src/components/fields/SchemaField.tsx b/src/components/fields/SchemaField.tsx
index ee7bd23ca5d034ef2234f0d909f6c16a03002db4..e99ef6aa8de4670cdf6e2225f1e612e80dd2255f 100644
--- a/src/components/fields/SchemaField.tsx
+++ b/src/components/fields/SchemaField.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useCallback, Component } from 'react';
 import {
   ADDITIONAL_PROPERTY_FLAG,
