diff --git a/dist/spectral.js b/dist/spectral.js
index 584249bc1b351cae0b1f3176b1ff8ffe06ed7175..c2b59b2c3d6059dddab255fbe4101124b442095a 100644
--- a/dist/spectral.js
+++ b/dist/spectral.js
@@ -63,7 +63,7 @@ class Spectral {
         return (await this.runWithResolved(target, opts)).results;
     }
     setRuleset(ruleset) {
-        this.ruleset = ruleset instanceof ruleset_1.Ruleset ? ruleset : new ruleset_1.Ruleset(ruleset);
+        this.ruleset = ruleset;
     }
     _generateUnrecognizedFormatError(document, formats) {
         return (0, generateDocumentWideResult_1.generateDocumentWideResult)(document, `The provided document does not match any of the registered formats [${formats
